### PR TITLE
Miscellaneoush Decomp 324 :)

### PR DESCRIPTION
**Pull request by Brotenko**
get fugd alex :)

### CODE REVIEW

File: src/battle/star/up_and_away/7952E0.c [hunk 1]
```diff
@@ -1,21 +1,60 @@
 #include "common.h"
+#include "battle/battle.h"
 
 #define NAMESPACE battle_star_up_and_away
 
-#include "common/UnkStarFuncs.inc.c"
+#include "common/StarPower.inc.c"
 
-#include "common/FadeBackgroundToBlack.inc.c"
+//INCLUDE_ASM(s32, "battle/star/up_and_away/7952E0", func_802A1518_7957F8);
```
Comment (ethteck) [inline src/battle/star/up_and_away/7952E0.c:8]: INCLUDE_ASM comment can go away here

File: src/battle/star/up_and_away/7952E0.c [hunk 2]
```diff
@@ -1,21 +1,60 @@
...
+ApiStatus func_802A1518_7957F8(ScriptInstance* script, s32 isInitialCall) {
+    if (isInitialCall != 0) {
```
Comment (ethteck) [inline src/battle/star/up_and_away/7952E0.c:10]: no != 0 for bools

File: src/battle/star/up_and_away/7952E0.c [hunk 3]
```diff
@@ -1,21 +1,60 @@
...
+        return ApiStatus_DONE2;
+    }
 
-INCLUDE_ASM(s32, "battle/star/up_and_away/7952E0", func_802A1494_795774);
+    return ApiStatus_BLOCK;
+}
 
-#include "common/SetNpcCollision32.inc.c"
+ApiStatus func_802A15B4_795894(ScriptInstance* script, s32 isInitialCall) {
+    if (isInitialCall != 0) {
```
Comment (ethteck) [inline src/battle/star/up_and_away/7952E0.c:29]: != 0

**Review by ethteck — CHANGES_REQUESTED** [review]
w00t

**Review by ethteck — APPROVED** [review]

File: src/battle/star/chill_out/7900D0.c [hunk 4]
```diff
@@ -2,20 +2,9 @@
...
-#include "common/FadeBackgroundToBlack.inc.c"
-
-#include "common/UnkBackgroundFunc2.inc.c"
-
-#include "common/UnkBackgroundFunc.inc.c"
-
-INCLUDE_ASM(s32, "battle/star/chill_out/7900D0", func_802A1494_790564);
-
-#include "common/SetNpcCollision32.inc.c"
+#include "common/StarPower.inc.c"
```
Comment (nanaian) [inline src/battle/star/chill_out/7900D0.c:7]: Awesome

**Review by nanaian — APPROVED** [review]

**Pull request merged**